README.md

```diff
@@ -1,2 +1,14 @@
 # Adapter
 this repo contains Adapter design pattern implementation using C#
+
+## The problem 
+ 
+![UML](Problem.gif)
+
+I have faced this problem in one of my previous projects. And as a result, I ended up solving the problem by using the 'Adapter design pattern'. The way that I did was by adding a class ,Adapter(), that plays as the adapter role.  
+
+
+
+### UML
+
+   ![UML](UML.png)
```

Comment:
Good UML! but the arrows and relationships between classes aren't that clear.
It would be better if you used interface as the same as the Adapter UML 
![UML](https://refactoring.guru/images/patterns/diagrams/adapter/structure-object-adapter-2x.png?id=03e8052e168c962d6bc3)

Reply:
Ok, I will make it better soon 👍

Reply:
UML has been updated 👍

Reply:
great job Ahmed!!